source/fundamentals/data-formats/document-data-format-pojo.txt

```diff
@@ -115,113 +66,133 @@ make sure to follow these guidelines:
 Configure the Driver for POJOs
 ------------------------------
 
-To set up the driver to store and retrieve POJOs, we need to specify:
+To configure the driver to use POJOs, you must specify the following
+components:
 
-- The ``PojoCodecProvider``, a codec provider that includes
-  :doc:`Codecs </fundamentals/data-formats/codecs>` that define how to
-  encode/decode the data between the POJO and MongoDB document, and which
-  POJO classes or packages that the codecs should apply to.
-- A ``CodecRegistry`` instance that contains the codecs and other related information.
-- A ``MongoDatabase`` or ``MongoCollection`` instance configured to use the
+- ``PojoCodecProvider``, a codec provider that includes
+  :ref:`Codecs <fundamentals-codecs>` that define how to
+  encode and decode the data between the POJO format and BSON. The
+  provider also specifies which POJO classes or packages that the codecs
+  apply to.
+- ``CodecRegistry`` instance that contains the codecs and other related information.
+- ``MongoDatabase`` or ``MongoCollection`` instance configured to use the
   ``CodecRegistry``.
-- A ``MongoCollection`` instance created with the POJO document class
+- ``MongoCollection`` instance that is created with the POJO document class
   bound to the ``TDocument`` generic type.
 
-Consult the following steps to see how to perform each of the configuration
-requirements:
+The following steps perform each of the configuration requirements to
+use the sample POJO defined in the preceding section:
```

Comment:
S: the wording here kinda implies that the step itself is the subject taking some action; you could reword so that the reader is the subject and the one taking action

```suggestion
Perform the following steps to meet the configuration requirements defined in the preceding section:
```